command/format.go

```diff
@@ -557,6 +557,9 @@ func (t TableFormatter) OutputSecret(ui cli.Ui, secret *api.Secret) error {
 				for _, constraint := range constraintSet.Any {
 					out = append(out, fmt.Sprintf("mfa_constraint_%s_%s_id %s %s", k, constraint.Type, hopeDelim, constraint.ID))
 					out = append(out, fmt.Sprintf("mfa_constraint_%s_%s_uses_passcode %s %t", k, constraint.Type, hopeDelim, constraint.UsesPasscode))
+					if constraint.Name != "" {
+						out = append(out, fmt.Sprintf("mfa_constraint_%s_%s_name %s %s", k, constraint.Type, hopeDelim, constraint.Name))
```

Comment:
Since the `mfa_constraint` prefix is repeatedly reused, how do you feel about abstracting it out into a const?

Reply:
Hmm, it is nice idea, though there is only three repeats and I am not expecting to be more for a while. And, this way, the output is more readable as a reader does not need to jump around to find the value of the constant. So, I am leaning toward keeping it this way for now. But, I don't have strong feeling about it. What do you think?

Reply:
This works for me!